components/sidebar_header.jsx

```diff
@@ -39,7 +39,7 @@ export default class SidebarHeader extends React.Component {
         if (!this.props.currentUser) {
             return {};
         }
-        const tutorialStep = PreferenceStore.getInt(Preferences.TUTORIAL_STEP, this.props.currentUser.id, 999);
+        const tutorialStep = PreferenceStore.getInt(Preferences.TUTORIAL_STEP, this.props.currentUser.id, TutorialSteps.FINISHED);
```

Comment:
We're on a process of eliminating the use of old flux store in favor of redux, so I'd suggest not to use `PreferenceStore`. Instead, we're good with:
```
import {get} from 'mattermost-redux/selectors/entities/preferences';
```

Reply:
Thanks for the great feedback @saturninoabril! I had expunged the `PreferenceStore` for all the connected components in this PR, but opted not to change the `SidebarHeader` and `TutorialTip` since doing so would require a larger-scale refactor of these files. Would you be amenable to expunging the `PreferenceStore` as part of a separate task to connect these components (or their parents, depending on the right way to refactor)?

Reply:
Sure, separate task is good for me.